docs/explore/what-is-redis/index-what-is-redis.mdx

```diff
@@ -1,8 +1,7 @@
 ---
 id: index-what-is-redis
-title: Redis in-memory database. How it works and why you should use it.
+title: 'Redis:In-memory database. How it works and Why you should use it?'
```

Comment:
```suggestion
title: 'Redis:In-memory database. How it works and Why you should use it'
```

suggest removing the question mark as this is a statement of why you should use redis - as in the article's going to tell you why.